python/lsst/ts/vanward/check_software_releases.py

```diff
@@ -174,6 +174,8 @@ def main(opts: argparse.Namespace) -> None:
                 continue
             line = line.strip()
             parts = line.split("=")
+            if len(parts) < 2:
```

Comment:
Perhaps a comment explaining why this check is necessary would be helpful.

Reply:
Added comment.